### ISSUE TITLE
Accessibility audit: Create a list of common WCAG tests

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#5291]
## Description
Create a list of plain language tests that we can run on the design system components. 

Some goals for this:
- The tests should be easy to understand and easy to run, even for non-technical audiences. 
    - The tests should be written in plain language
    - The tests should be written in gherkin-style format. ("When this happens, you should experience this")
- It should be easy to share tests across multiple components
    - The tests should be written in a generic way whenever possible. 
    - The tests should have some method that makes the tests easy to reference (For example, a reference number like "2.4.6.1" could indicate the first test for WCAG 2.4.6)

## Acceptance Criteria
- [x] We have a list of tests that we can easily pull into the list of tests to perform on a component.

**Comment by amyleadem** [comment]
## Updates
- 02/09/2026: Piloting a library of common tests by creating tests for  the va-link componnet
